gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// There is no cache slug for dev.
+var cacheSlugDev = '';
 var generatedTargetDir = path.join(
   'third_party', 'generated',
-  isMinificationNeeded ? 'prod' : 'dev');
+  isMinificationNeeded ? 'prod' : cacheSlugDev);
```

Comment:
Oh, sorry -- I seem to have missed this comment, or read an older version of it.

Yep, sounds like there's a good reason for this. Can you add a (long) comment explaining in detail why we choose '' (to someone who's never heard of cache slugs)? That would make this more maintainable going forward.

Thanks!